core/src/test/resources/idContainsGeoJson.json

```diff
@@ -0,0 +1 @@
+{"type":"Feature","id":0,"properties":{"zipcode":"00101","name":"A"},"geometry":{"type":"Polygon","coordinates":[[[0.0,0.0],[10.0,0.0],[10.0,10.0],[0.0,10.0],[0.0,0.0]]]}}
```

Comment:
Is this same as testContainsId.json? Can you please remove all unused json files? Also please make sure,  they all use a consistent name such as "testContainsId".